src/cli.js

```diff
@@ -106,25 +90,26 @@ export const run = (launchFilePath: string): void => {
     );
 
     const comments = ((args.comments: any): string)
-        .split(",")
+        .split(" ")
         .filter((c) => !!c);
     const ignoreGlobs = ((args.ignore: any): string)
-        .split(",")
+        .split(";")
         .filter((c) => !!c);
 
-    const ignoreFileGlobs = args.noIgnoreFile
+    const ignoreFiles = args.noIgnoreFile
         ? []
-        : ignoreFilesToExcludes((args.ignoreFiles: any));
-    const excludeGlobs = [...ignoreGlobs, ...ignoreFileGlobs];
+        : ((args.ignoreFiles: any): string).split(",").filter((c) => !!c);
 
     // Make sure we have something to search, so default to current working
     // directory if no globs are given.
-    const includeGlobs = args._ && args._.length > 0 ? args._ : [process.cwd()];
+    const includeGlobs =
+        args._ && args._.length > 0 ? args._ : [`${process.cwd()}/**`];
```

Comment:
Made the default an actual glob instead of using "ignore file" syntax as a proxy for globs.